ui/app/models/config-ui/message.js

```diff
@@ -89,7 +89,7 @@ export default class MessageModel extends Model {
 
   // date helpers
   get isStartTimeAfterToday() {
-    return isAfter(this.startTime, new Date());
+    return isAfter(parseISO(this.startTime), new Date());
```

Comment:
Updated to use parseISO since date-fns doesn't accept strings as arguments. 
<img width="919" alt="Screenshot 2024-01-08 at 11 16 05 AM" src="https://github.com/hashicorp/vault/assets/30884335/ff2ca16b-024b-4a7d-aae7-1586cdc7a3ec">